src/pages/Trade/Trade.tsx

```diff
@@ -18,30 +18,40 @@ import CreateOrderCard from './components/CreateOrderCard';
 import OrderBookCard from './components/OrderBookCard';
 import BlurBackground from './components/BlurBackground';
 
-import { getSynthPair, setSynthPair, SynthPair } from 'ducks/synths';
+import { getSynthPair, setSynthPair, getAvailableSynthsMap } from 'ducks/synths';
 
-type StateProps = {
-	synthPair: SynthPair;
-};
+const mapStateToProps = (state: RootState) => ({
+	synthPair: getSynthPair(state),
+	synthsMap: getAvailableSynthsMap(state),
+});
 
-type DispatchProps = {
-	setSynthPair: typeof setSynthPair;
+const mapDispatchToProps = {
+	setSynthPair,
 };
 
-type Props = RouteComponentProps<{
-	baseCurrencyKey: CurrencyKey;
-	quoteCurrencyKey: CurrencyKey;
-}>;
+const connector = connect(mapStateToProps, mapDispatchToProps);
+
+type PropsFromRedux = ConnectedProps<typeof connector>;
```

Comment:
@clementbalestrat this is the new pattern for typing redux ^ so much shorter and better!

Reply:
oh cool, love it